build-tools/installers/create-installers.targets

```diff
@@ -186,6 +186,14 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\java_runtime_fastdev.jar" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\java_runtime.dex" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\java_runtime_fastdev.dex" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\manifestmerger\common.jar" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\manifestmerger\guava.jar" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\manifestmerger\kotlin-stdlib-jdk8.jar" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\manifestmerger\kotlin-stdlib.jar" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\manifestmerger\kxml2.jar" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\manifestmerger\manifest-merger.jar" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\manifestmerger\sdk-common.jar" />
+    <_MSBuildFiles Include="$(MSBuildSrcDir)\manifestmerger\sdklib.jar" />
```

Comment:
You should use the trick I used with r8.jar: https://github.com/xamarin/xamarin-android/blob/fbe4b44c7b40dcaacdecaed834c631b15490cc52/src/r8/build.gradle#L19-L28

I made a gradle project that brings in the packages, and this `zipTree` code combines them all into a self-contained jar file.